### PR TITLE
#3 Added check for correct number of command-line arguments

### LINKED ISSUE #3: Exceptions aren't very friendly if you miss an argument
I noticed that if you try to run the script without enough arguments you get an error message that you have to think about a bit before you figure out what might be wrong:

```
$ ./rename_files.py
Traceback (most recent call last):
  File "./rename_files.py", line 81, in <module>
    main()
  File "./rename_files.py", line 24, in main
    string_to_replace, replacement_string, path_of_files = get_input()
  File "./rename_files.py", line 37, in get_input
    string_to_replace = sys.argv[1]
IndexError: list index out of range
```

That's not terrible, but you could maybe make the exception a little friendlier than an `IndexError` if you did a check that `len(sys.argv)` is `3` in `get_input` and threw an ordinary `Exception` with a nicer error message.

This is just a minor thing. If you are fine with the message the way it is, that's cool with me.

### PR DESCRIPTION
**Pull request by kimlew**
Resolved #3
@kcharter As you suggested, I've added a check for the correct number of command-line arguments in get_input().

### CODE REVIEW

File: rename_files.py [hunk 1]
```diff
@@ -33,14 +33,22 @@ def main():
...
     replacement_string = sys.argv[2]
     path_of_files = sys.argv[3]
 
-    print('String to replace: ', string_to_replace)
-    print('Replacement string: ', replacement_string)
-    print('Folder path: ', path_of_files)
+    print(' String to replace: ', string_to_replace)
+    print(' Replacement string: ', replacement_string)
+    print(' Folder path:', path_of_files)
     print()
```
Comment (kcharter) [inline rename_files.py:52]: You've changed these three lines with the print statements in different ways in your two PRs. Obviously the changes aren't a big deal but in general it's a good idea to keep a PR focused on a single purpose, where all the changes contribute to that purpose.

No need to change anything here, just wanted to point it out.

File: rename_files.py [hunk 2]
```diff
@@ -33,14 +33,22 @@ def main():
...
 
 def get_input():
+    print(" You entered", len(sys.argv)-1, "arguments at the command line.")
+
+    if len(sys.argv) != 4:
+        # IndexError: list index out of range
+        print(" Error: Wrong number of arguments. Enter 3 arguments: 1. "
+              "string to replace 2. replacement string 3. path for files ")
+        sys.exit(1)
+
```
Comment (kcharter) [inline rename_files.py:43]: This works but it has the drawback that a decision to exit the program is buried in a function, and most people advise putting that kind of thing in the main block.

To do that, you'd change your check to raise an exception with a nicer error message:

```
if len(sys.argv) != 4:
    raise Exception("Error: Wrong number of arguments ...")
```

If you leave it like that, when the user gives the wrong number of arguments, there will still be a stack trace, but the message will explain the problem in terms that make more sense to the user.

**Review by kcharter — CHANGES_REQUESTED** [review]
Thinking about it, it's probably better to adopt the 'Pythonic' practice of raising an exception in `get_input`.

File: rename_files.py (hunk 1 above)
Comment (kimlew) [reply rename_files.py:52]: Ok. Changed it to Pythonic way to handle IndexError.

**Review by kcharter — APPROVED** [review]

**Pull request merged**